### DISCUSSION TITLE
Serving user customizable templates

### DISCUSSION THREAD

**Discussion by robeng1** [seaweedfs/seaweedfs#2148] [Q&A] [answered]
I'm trying to implement something similar to shopify's storefront renderer https://shopify.engineering/how-shopify-reduced-storefront-response-times-rewrite and I wanted to ask if seaweed will be ideal for managing the templates

**Comment by chrislusf** [comment, marked as answer]
Should be. You can `weed mount` and use it just as a local folder.

**Reply by robeng1** [reply]
Okay. Thanks very much
